src/benchcab/workdir.py

```diff
@@ -24,6 +24,9 @@ def clean_submission_files():
     if internal.RUN_DIR.exists():
         shutil.rmtree(internal.RUN_DIR)
 
+    if internal.STATE_DIR.exists():
+        shutil.rmtree(internal.STATE_DIR)
```

Comment:
We should ensure that STATE_DIR is actually set. Some systems may treat a blank as valid, then the next command will attempt to wipe the root...

I've seen it happen, it ain't pretty.

Reply:
I agree but I'm not sure how to address this. `STATE_DIR` is a constant `pathlib.Path` object and is not possible for the object to be 'unset' (by default `Path()` evaluates to the current working directory).

The underlying issue to this might be the approach we are taking when doing clean up. A safer way could be to archive run directories instead of deleting them recursively, and have an explicit command to do the deletion. This is similar to how `payu sweep` works. This way the deletion would not depend on a variable which could be misconfigured.